vita3k/main.cpp

```diff
@@ -213,6 +214,10 @@ int main(int argc, char *argv[]) {
     if (host.io.title_id.find("PCS") != std::string::npos)
         host.app_sku_flag = get_license_sku_flag(host, host.app_content_id);
 
+    // Pre-Compile Shader only for glsl, spriv is broken
+    if (!host.cfg.spirv_shader)
+        renderer::gl::pre_compile_programs(static_cast<renderer::gl::GLState &>(*host.renderer), host.base_path.c_str(), host.io.title_id.c_str(), host.renderer->shaders_count_compiled);
```

Comment:
Rather then making a shader count compiled variable very unrelated to host, why not just stored it in renderer?

Reply:
i no understund yoiur ask here, is already inside renderer state here